packages/accordions/src/elements/accordion/components/Label.tsx

```diff
@@ -10,7 +10,7 @@ import { StyledButton } from '../../../styled';
 import { useAccordionContext, useHeaderContext, useSectionContext } from '../../../utils';
 
 /**
- * @extends HTMLButtonElement
+ * @extends ButtonHTMLAttributes<HTMLButtonElement>
  */
 export const Label = forwardRef<HTMLButtonElement, HTMLAttributes<HTMLButtonElement>>(
```

Comment:
Same as the comment below, we may want to update `HTMLAttributes` to `ButtonHTMLAttributes` here

Reply:
The extends have all been removed for the time being.